package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "simple-icons",
-  "version": "1.9.19",
+  "version": "1.9.13",
```

Comment:
Set it to 1.9.19, as in current develop in origin repository. 
Commit only this one change and push.

Reply:
There are multiple ways to resolve conflict with this file.

As in your work you changed `package.json`, and in the mean time it was also changed on the branch you want to merge to - possibilities are, not limited to:

1. set this line to the value that *already is in develop* (i.e. "version": "1.9.19") (will be discovered as no change to merge)
2. restore it to the value *you started with*, and currently your branch starts at a853feb, and the line is "version": "1.9.18" there (will be discovered as no change done by you).

There are other solutions but I think _1_ is the easiest.

Reply:
Awesome! I will change it to version 1.9.19. Thank you for the explanation. 🙏